tests/test_efdUtils.py

```diff
@@ -51,12 +54,24 @@
 except ImportError:
     HAS_EFD_CLIENT = False
 
+# Use mode="none" to run tests for normal operation.
+# To update files or generate new ones, make sure you have a working
+# connection to lsst-schema-registry-efd.ncsa.illinois.edu
```

Comment:
Is this address true?  I'm a bit surprised to still see nasa dependency.

Reply:
Oh nvm... I see you corrected this in a later commit.  I guess that commit could be squashed with the first one.